packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -1188,24 +1351,74 @@ export class EditableGrid extends PureComponent<EditableGridProps, EditableGridS
                     />
                 </div>
                 {allowAdd && this.getControlsPlacement() !== 'top' && this.renderAddRowsControl('bottom')}
+            </>
+        );
+
+        const bulkUpdateContent = (
```

Comment:
I think by making this into a `const` member it will cause the component to be rendered even when `showBulkUpdate` is not true, which will be a little less efficient in the non-tabbed case, but I see this allows for reuse, so perhaps this is a case where using a `this.renderBulkUpdate()` method would be best?